webpack/assets/javascripts/foreman_tools.js

```diff
@@ -1,3 +1,7 @@
+'use strict';
+
+const $ = require('jquery');
```

Comment:
ES6 modules are always in strict mode so no need to declare it.

@dLobatog You can also use the import statement instead of require

```js
import $ from 'jquery';
```